gitkraken-client/single-sign-on.md

```diff
@@ -291,4 +291,92 @@ Step 10: Select “I am an Okta customer adding an internal app” from option m
+<img src="/wp-content/uploads/sso-pingidentity-6-2.png" class="img-bordered img-responsive center">
+
+7.	Select *Identities* to add some users. Once you are done adding them, go ahead and click on *Groups*, then click on the plus button to add a group. (Make sure to add users with their email addresses).
+
+<img src="/wp-content/uploads/sso-pingidentity-7.png" class="img-bordered img-responsive center">
+<img src="/wp-content/uploads/sso-pingidentity-7-2.png" class="img-bordered img-responsive center">
+
+8.	Select *Groups*, then click on the plus button to add a group. Once you have that, you can add the users to your group.
+
+<img src="/wp-content/uploads/sso-pingidentity-8.png" class="img-bordered img-responsive center">
+
+9.	Select *Connections*.
+
+<img src="/wp-content/uploads/sso-pingidentity-9.png" class="img-bordered img-responsive center">
+
+10.	Click on the plus button.
+
+<img src="/wp-content/uploads/sso-pingidentity-10.png" class="img-bordered img-responsive center">
+
+11.	Enter a name for your application, then select *SAML Application*. Next click on the *Configure* button which appears once you select your application type.
+
+<img src="/wp-content/uploads/sso-pingidentity-11.png" class="img-bordered img-responsive center">
+
+12.	Select *Manually Enter*. Type in the URL for *ACS URLs* and *Entity ID*, then click on *Save*.
+(URL: `https://api.gitkraken.com/oauth/sso/callback`))
+
+<img src="/wp-content/uploads/sso-pingidentity-12.png" class="img-bordered img-responsive center">
+
+13.	Click on the toggle button so the users would have access to your application.
+
+<img src="/wp-content/uploads/sso-pingidentity-13.png" class="img-bordered img-responsive center">
+
+14.	Click on *Attributes* then add email as your new attribute.
+
+<img src="/wp-content/uploads/sso-pingidentity-14.png" class="img-bordered img-responsive center">
+<img src="/wp-content/uploads/sso-pingidentity-14-2.png" class="img-bordered img-responsive center">
+
+15. Time to add the group we created.
+
+<img src="/wp-content/uploads/sso-pingidentity-15.png" class="img-bordered img-responsive center">
+
+16.	Click on the button below.
+
+<img src="/wp-content/uploads/sso-pingidentity-16.png" class="img-bordered img-responsive center">
+
+17.	Click on the plus icon to add the group, then click on *Save*.
+
+<img src="/wp-content/uploads/sso-pingidentity-17.png" class="img-bordered img-responsive center">
+<img src="/wp-content/uploads/sso-pingidentity-17-2.png" class="img-bordered img-responsive center">
+
+18.	Go to the *Configuration* tab to copy your *IDP Metadata URL* and download your metadata (*Download Metadata* button).
+
+<img src="/wp-content/uploads/sso-pingidentity-18.png" class="img-bordered img-responsive center">
+
+19.	Go to your organization and click on *SSO* then click on *Enable SSO*. Type in your domain in *Organization Domain Name* with *.com* . Then click on *Configure SSO Connection*.
```

Comment:
Update to:

19.	On app.gitkraken.com, go to your organization, click on *SSO*, and then click on *Enable SSO*. Type in your domain in *Organization Domain Name* with *.com* . Then click on *Configure SSO Connection*.